README.md

```diff
@@ -32,19 +32,45 @@ to get you started.
 
 Releases of the code can be installed with pip as usual:
 
-    pip install glass
+```sh
+pip install glass
+```
```

Comment:
AFAICT, this renders exactly the same on the web, but makes it slightly uglier in the plain text, so why change it?

Reply:
Because indenting only works for `sh`, the `bibtex` syntax highlighting only works with code fencing. In general, we should be using code fencing https://github.com/updownpress/markdown-lint/blob/master/rules/040-fenced-code-language.md